### DISCUSSION TITLE
How to Compare Two Vector Graphs in dxf use ezdxf ?

### DISCUSSION THREAD

**Discussion by zhuochengs** [mozman/ezdxf#869] [Q&A]
Thank you, this is not a bug

I have not studied CAD or Python professionally, so I would like to describe my ideas. Thank you again.

I now have a vector graph A, a set of vector graphs S (which contains many vector graphs), and I want to find some sets of graphs X from S that share some similarities with vector graph A (assuming N vector graphs meet my needs)
![微信图片_20230424112048](https://user-images.githubusercontent.com/17957646/233892953-301c07ab-559e-4766-b105-3de2dedbba1a.png)

As shown in the following figure, the cyan image is X, and the pink image is A. Therefore, the vector images in set X can meet the following requirements:

Assuming that two graphs share the center of a circle, vector graph A remains stationary, X rotates, and at a certain moment, X can completely cover graph A; Namely:

1. The maximum circumscribed radius of X cMaxX>cMaxA

2. The minimum circumscribed radius of X cMinX>cMinA

3. The vector graph data set of X should not intersect with the vector graph data set of A, that is, there is no intersection point

I want to implement the above ideas and inquire about the key judgments. Have you provided them in your document?

**Comment by mozman** [comment]
Hey, I'm not sure if this is the right forum for your question. It seems like it might be more appropriate for [StackOverflow](https://stackoverflow.com/), where you might find more experts in that area.

**Reply by zhuochengs** [reply]
thank you!
